resources.whatwg.org/build/deploy.sh

```diff
@@ -93,9 +93,11 @@ MATCHES=$(
   perl -ne '$/ = "\n\n"; print "$_" if (/\ban\s+(<[^>]*>)*(?!(L\b|http|https|href|hgroup|rb|rp|rt|rtc|li|xml|svg|svgmatrix|hour|hr|xhtml|xslt|xbl|nntp|mpeg|m[ions]|mtext|merror|h[1-6]|xmlns|xpath|s|x|sgml|huang|srgb|rgb|rsa|rst_stream|only|option|optgroup)\b|html)[b-df-hj-np-tv-z]/si or /\b(?<![<\/;])a\s+(?!<!--grammar-check-override-->)(<[^>]*>)*(?!&gt|one)(?:(L\b|http|https|href|hgroup|rt|rp|li|xml|svg|svgmatrix|hour|hr|xhtml|xslt|xbl|nntp|mpeg|m[ions]|mtext|merror|h[1-6]|xmlns|xpath|s|x|sgml|huang|srgb|rsa|only|option|optgroup)\b|html|[aeio])/si)' "$INPUT_FILE" | perl -lpe 'print "\nPossible grammar problem: \"a\" instead of \"an\" or vice versa (to override, use e.g. \"a <!--grammar-check-override-->apple\"):" if $. == 1'
   grep -ni 'and/or' "$INPUT_FILE" | perl -lpe 'print "\nOccurrences of making Ms2ger unhappy and/or annoyed:" if $. == 1'
   grep -niE '\s+$' "$INPUT_FILE" | perl -lpe 'print "\nTrailing whitespace:" if $. == 1'
+  grep -P '\t' "$INPUT_FILE" | perl -lpe 'print "\nTab:" if $. == 1'
 )
 if [ -n "$MATCHES" ]; then
   echo "$MATCHES"
+  exit 1
```

Comment:
We should check all specifications to ensure they don't already fail for a reason we might not be able to fix.

Reply:
Failures from https://travis-ci.org/whatwg:

- https://travis-ci.org/whatwg/xhr unicorns-are-teh-awesome, a HTML document (real errors)
- https://travis-ci.org/whatwg/mimesniff an mp3 (script needs fixing)
- https://travis-ci.org/whatwg/compat an RGBA (script needs fixing)

I'll fix the script for mp3/rgba

Reply:
unicorns-are-teh-awesome is an intentional misspelling, but I guess there's an opt-out for those now?

Reply:
There's not.

Reply:
Should we add one?

Reply:
I ended up fixing the XHR issue. I'd be okay with not adding anything until there's a more justified need.

Reply:
Cool. Does this need anything else before it can be merged, then?